### PR TITLE
admin action to exercises of user

### PR DESCRIPTION
**Pull request by wowKimsohyun17**
## Related Tickets:
- [#30233](https://edu-redmine.sun-asterisk.vn/issues/30233)

## WHAT this PR does?
admin review exercises

## Notes Impacted Areas:
- N/A.

## Attachments:
as

### CODE REVIEW

**Comment by wowKimsohyun17** [comment]
ready

**Comment by wowKimsohyun17** [comment]
ready

**Review by huyenttm-0901 — COMMENTED** [review]
conflict

**Comment by wowKimsohyun17** [comment]
ready

File: resources/lang/en/label.php [hunk 1]
```diff
@@ -162,4 +162,17 @@
...
+    'table_exercise' => 'Bài tập',
+    'table_lesson' => 'Danh sách bài học',
+    'lesson_title' => 'Bài học',
+    'all' => 'Tất cả',
+    'table_course' => 'Danh sách khóa học',
+    'course_title' => 'Khóa học',
+    'table_exercise_user' => 'Danh sách bài tập',
+    'students' => 'Danh sách học sinh',
+    'exercise_user_title' => 'Bài tập',
+    'table-name' => 'Học sinh',
```
Comment (huyenttm-0901) [inline resources/lang/en/label.php]: lần lôn en với vi này

File: resources/lang/vi/label.php [hunk 2]
```diff
@@ -169,4 +169,10 @@
     'enter_link_exercise' => 'Điền câu trả lời của bạn',
     'test' => 'Bài tập',
     'need_finish' => 'Vui lòng hoàn thành các bài học trước để mở khóa bài học này',
+    'table_exercise_user' => 'Danh sách bài tập',
+    'students' => 'Danh sách học sinh',
+    'exercise_user_title' => 'Bài tập',
+    'table-name' => 'Học sinh',
+    'dashboard' => 'Dashboard',
+    'f8' => 'F8',
```
Comment (huyenttm-0901) [inline resources/lang/vi/label.php]: file này thiếu nhé

**Comment by wowKimsohyun17** [comment]
ready

**Pull request merged**